src/ui/buttons/buttonDesigns/control.jsx

```diff
@@ -0,0 +1,9 @@
+/* @flow */
+/** @jsx node */
+import { node, type ChildType } from 'jsx-pragmatic/src';
+
+export function ControlDesignComponent(experimentId : string) : ChildType {
+    return (
+        <div data-design-experiment={ experimentId } />
```

Comment:
I like the data attribute approach 👍

Reply:
Yeah it should be better. Only worry I have with logging is how we ignore the resize cases for logs. Right now the designs will be removed whenever the button doesn't match the design requirements. We should send some signal in the logs, but not sure how to go about it